music21/voiceLeading.py

```diff
@@ -1176,6 +1180,9 @@ def leapNotSetWithStep(self) -> bool:
         if self.noMotion():
             return False
 
+        # .generic could only be None if intervals were constructed without diatonic components
+        assert self.hIntervals[0].generic is not None
+        assert self.hIntervals[1].generic is not None
```

Comment:
Same here.  Work around or disable.  I'm okay assigning to a singleton GenericInterval in the class definition if need be.  

Users of an object should not be responsible for checking corner cases of the definition of a lower-level object.  And VoiceLeadingQuartet is a user of Interval.